### ISSUE TITLE
[Two-handed Weapon Specialization]

### ISSUE THREAD

**Issue by kbinside** [tobmaps/TrinityCore#137]
Still occurring - Issue #1276 -> https://github.com/TrinityCore/TrinityCore/issues/1276

**Comment by tobmaps** [comment]
it already fixed for a long time ago in my fork

**Comment by kbinside** [comment]
yeah some said so but its still does not increase damage, tested with meele swings (even visual bug still occures).

**Comment by tobmaps** [comment]
then why it increases damage (visual and real) for me?

**Comment by kbinside** [comment]
if it would work for me, why should i write here?

**Comment by tobmaps** [comment]
you should check if you correctly installed [p60]

**Comment by kbinside** [comment]
question is, it was added to any revision?

**Comment by tobmaps** [comment]
[p60] already included in my fork for a long time.

**Comment by Craghorn** [comment]
Analog bug with pally talent.

**Comment by Craghorn** [comment]
Specialization work's fine almolst with all attack.
Only exept pally Judgement's (

**Comment by kbinside** [comment]
and how you tested that "only" for?

**Comment by Craghorn** [comment]
... just unlearn all, and learn this auras, use Judgement

**Comment by Nedj** [comment]
Well then it's anyway inaccurate to be "only" this.

On the topic, the patch might not work correctly for you if you missed to implement the database part of the patch, as Tobmaps stated above.
